selfapi.js

```diff
@@ -367,6 +372,21 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  toAPIIndex: function (basePath) {
+    var fullPath = normalizePath(this.path, basePath) || '/';
+    return function () {
+      var routes = {};
+      Object.keys(this.children).forEach(function (child) {
+        if (child == "null") {
+          return;
```

Comment:
I don't have an error but a route `null: "/api/v1/null"` gets added.

Code to generate it is [here](https://framagit.org/phyks/kresus/commit/ff475a9c754ed9ae1ecf7c126635acfc2a25adc8).